omnibus-sepolia.toml

```diff
@@ -132,7 +141,292 @@ func = "setPoolConfiguration"
+
+[setting.maintenance_margin_scalar]
+defaultValue = "<%= parseEther('0.5') %>"
+
+[setting.max_initial_margin_ratio]
+defaultValue = "<%= parseEther('0.9') %>"
+
+[setting.liquidation_reward_percent]
+defaultValue = "<%= parseEther('0.0001') %>"
+
+[setting.liquidation_limit_scalar]
+defaultValue = "<%= parseEther('1') %>"
+
+[setting.liquidation_max_pd]
+defaultValue = "<%= parseEther('0.0006') %>"
+
+[setting.liquidation_window_duration]
+defaultValue = "36"
+
+# --- Provision bfp market --- #
+#
+#
+
+[provision.bfp_market_factory]
+source = "<%= settings.bfp_market_package %>"
+targetPreset = "<%= settings.target_preset %>"
+options.coreProxyOwner = "<%= settings.owner %>"
+options.owner = "<%= settings.owner %>"
+options.salt = "<%= settings.salt %>"
+options.synthetixPackage = "<%= settings.snx_package %>@<%= settings.target_preset %>"
+options.synthetixPreset = "<%= settings.target_preset %>"
+depends = ["provision.system"]
+
+# --- Provision fake collaterals --- #
+#
+#
+
+# Create a mintable fake ETH for collateral
+#
+
+[provision.weth_mock_collateral]
+source = "mintable-token:1.8"
+options.name = "Fake WETH"
+options.symbol = "fWETH"
+options.owner = "<%= settings.owner %>"
+options.decimals = "18"
+
+# Set weth_address so that Core is configured with the fake WETH.
+#
+[setting.weth_address]
+defaultValue = "<%= console.log('weth_mock_collateral',weth_mock_collateral) || weth_mock_collateral.MintableToken.address %>"
+
+# --- Configure bfp --- #
+#
+#
+
+# Set pyth
+#
+
+[invoke.BfpMarketProxy_setPyth]
+target = ["bfp_market_factory.BfpMarketProxy"]
+fromCall.func = "owner"
+func = "setPyth"
+args = ["<%= settings.pyth_price_verification_address %>"]
+
+# --- Add Rewards distributors --- #
+#
+[invoke.BfpMarketProxy_createRewardDistributor_weth]
+target = ["bfp_market_factory.BfpMarketProxy"]
+fromCall.func = "owner"
+func = "createRewardDistributor"
+extra.weth_rewards_distributor.event = "RewardDistributorCreated"
+extra.weth_rewards_distributor.arg = 0
+
+[[invoke.BfpMarketProxy_createRewardDistributor_weth.args]]
+poolId = "<%= settings.sc_pool_id %>"
+name = "weth RewardDistributor"
+token = "<%= settings.weth_address %>"
+collateralTypes = [
+    "<%= settings.snx_address %>",
+    "<%= imports.wsteth.contracts.Token.address %>",
+    "<%= imports.susde.contracts.Token.address %>",
+]
+
+# --- Global Market configuration --- #
+#
+#
+
+[invoke.BfpMarketProxy_setMarketConfiguration]
+target = ["bfp_market_factory.BfpMarketProxy"]
+fromCall.func = "owner"
+func = "setMarketConfiguration"
+
+[[invoke.BfpMarketProxy_setMarketConfiguration.args]]
+pythPublishTimeMin = 12
+pythPublishTimeMax = 60
+minOrderAge = 12
+maxOrderAge = 60
+minKeeperFeeUsd = "<%= settings.min_keeper_fee_usd %>"
+maxKeeperFeeUsd = "<%= settings.max_keeper_fee_usd %>"
+keeperProfitMarginUsd = "<%= settings.keeper_profit_margin_usd %>"
+keeperProfitMarginPercent = "<%= settings.keeper_profit_margin_percent %>"
+keeperSettlementGasUnits = 1200000
```

Comment:
yeah i will reconfigure these once i can run some trades on sepolia forks hopefully. So I dont know if there is a way, aside from my post it notes, to have these parameters in review mode 😅